cmd/cli/triggers.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"time"
+
+	"github.com/moira-alert/moira"
+)
+
+func handleRemoveTriggersByPrefix(logger moira.Logger, database moira.Database, prefix string) error {
+	triggers, err := database.GetTriggerIDsByPrefix(prefix)
```

Comment:
does func return triggers by prefix in id, doesnt it?

Reply:
yes, noted in [comment](https://github.com/moira-alert/moira/pull/790/commits/ecd22470e6d393f0cfa7b53fbee551a53c199231#diff-f463a96da8e66f155de89be8fc4d0ecde2eb0f6e556a4cc5642b0d3c0e5df5deR153)

Reply:
think that can be misunderstanding, lets think about naming, cause notion prefix in Moira already use, ya?

Reply:
do you mean prefix tree?

Reply:
yep, as least

Reply:
I don't agree about misunderstanding, prefix is common term for programming. So prefix means some symbols at start of string. There is no reference to tree in method name or comment of GetTriggerIDsByPrefix

Reply:
renamed for GetTriggerIDsStartWith(prefix string)